connectors/uplynk/src/main/java/com/theoplayer/android/connector/uplynk/UplynkSsaiDescription.kt

```diff
@@ -0,0 +1,89 @@
+package com.theoplayer.android.connector.uplynk
+
+import com.theoplayer.android.api.source.ssai.CustomSsaiDescription
+import kotlinx.serialization.Serializable
+
+/**
+ * The configuration for server-side ad insertion using the [UplynkConnector].
+ */
+@Serializable
+class UplynkSsaiDescription(
+    val prefix: String?,
+    val assetIds: List<String>,
+    val externalId: List<String>,
+    val userId: String?,
+    val preplayParameters: LinkedHashMap<String, String>
```

Comment:
Add default values for all properties. That way, customers can call `UplynkSsaiDescription()` instead of `UplynkSsaiDescription.Builder().build()`. 😉